Update Tournaments wiki article `TEMPLATE` with numbered mappool list (#6496)
ppy/osu-wiki: wiki/Tournaments/TEMPLATE.md

**[Download the mappack here! (SIZE)](LINK)**

- ModType
  1. [BeatmapArtist - BeatmapTitle (BeatmapCreator) [BeatmapDifficulty]](BeatmapLink)
- Tiebreaker
  1. **Beatmap**

<!-- tiebreaker beatmaps must be bolded 

> If the mappool doesn't use a standard ModType pool structure, alternative titles may be used in its place. -->

Reviewer: there is a very intellectual reason for this change but i can't quite put it to words

```suggestion
If the mappool doesn't use a standard ModType pool structure, alternative titles may be used instead. -->
```
